### ISSUE TITLE
Error 13 on loading backups from USB device.

### ISSUE THREAD

**Issue by BombchuLink** [FIX94/Nintendont#826]
USB device: 
     Seagate 1TB Expansion Portable Device (STEA1000400)
Format: 
     FAT32; 32K cluster (primary partition)
Nintendont Version: 
     Latest Release
Wii Version: 
     4.3
Issue:
     On loading games from USB in Nintendont, It displays error 13, adding that it cannot load the USB device. The error happens after selecting the game title and asking it to begin loading. I do not have issues loading .wbfs files (via Wiiflow Lite) on the same harddrive. I also do not have issue loading homebrew applications that work with USB devices on this USB device.

**Comment by carnage702** [comment]
nintendont uses a stripdown usb code due to memory limitations so the compatibility you have with other homebrews with some usb drives may vary.

first what version of nintendont are you using?(number on top left of the screen)

is the hdd partitioned? nintendont doesnt play well with partitions and while its possible even hidden partitions sometimes give it issues.

other times is just the hdd being too fragmented is enough for nintendont to not like it which needs a slow format or a defragmentation.

**Comment by BombchuLink** [comment]
I have just updated to v6.489 (Wii)

The HHD is not partitioned, there is only one partition, and it is FAT32; 32K cluster

The error message now says ``WARNING: USB FAT device could not be opened`` Instead of the error previously stated.

**Comment by carnage702** [comment]
> 
> 
> I have just updated to v6.489 (Wii)
> 
> The HHD is not partitioned, there is only one partition, and it is FAT32; 32K cluster
> 
> The error message now says `WARNING: USB FAT device could not be opened` Instead of the error previously stated.

take a picture of your games path like root inside games folder and then inside an individual game

**Comment by BombchuLink** [comment]
USB

![image](https://user-images.githubusercontent.com/34423159/98981470-d4fcc700-24d2-11eb-859b-ea7cd8500fc0.png)

Games:
![image](https://user-images.githubusercontent.com/34423159/98981544-f1006880-24d2-11eb-9b4d-a48f7e626c54.png)

ISO
![image](https://user-images.githubusercontent.com/34423159/98981566-fb226700-24d2-11eb-9a34-da9081d5df64.png)

**Comment by carnage702** [comment]
path seems fine just wondering what settings do you have set to on on nintendont? any chance for a picture? also do you have any 0kb files on your saves folder?

**Comment by BombchuLink** [comment]
Settings:
![IMG_20201112_104224](https://user-images.githubusercontent.com/34423159/98982528-66b90400-24d4-11eb-8dcc-271166cf1968.jpg)


And no, there are two .raw files and they both are 2MB in the save folder.

**Comment by carnage702** [comment]
well my only guess is the partition table your hdd uses nintendont likes MBr partitions and its very iffy with GPt partition table, not sure what your hdd uses tbh.

if all else fails a slow format with a decent tool should do the trick, unless your hdd just ends up imcompatible with nintendont.

**Comment by BombchuLink** [comment]
The Harddrive has a MBR partition so that's not the issue. 

I went and formatted the harddrive again, and it magically started working. I just used ``mkfs.fat`` and nintendon't (as well as every other homebrew application) liked it, so I guess I'm good.

Morale of the story is to not use ``HPUSBDisk.exe`` to format drives (even though some guy on GBATemp said to) and to use ``mkfs.fat`` instead.

Thanks for the assistance Carnage, and hopefully anybody with the same error can get results from reading this.
